buildbot/osuosl/master/config/builders.py

```diff
@@ -414,22 +414,34 @@
                     checkout_lld=False,
                     extra_cmake_args=["-DLLVM_TARGETS_TO_BUILD='AArch64'"])},
 
-    ## AArch64 check-all + LLD + test-suite 2-stage
+    # AArch64 2 staage build with lld, flang, compiler-rt, test-suite and SVE/SME
+    # mlir integration tests.
     {'name' : "clang-aarch64-lld-2stage",
     'tags'  : ["lld"],
     'workernames' : ["linaro-clang-aarch64-lld-2stage"],
     'builddir':"clang-aarch64-lld-2stage",
     'factory' : ClangBuilder.getClangCMakeBuildFactory(
                 clean=True,
+                checkout_flang=True,
+                checkout_lld=True,
                 useTwoStage=True,
+                testStage1=False,
```

Comment:
Stage 1 tests are now being skipped because with have similar bots that run them, right?

Reply:
No, I just copied it over thinking it was a default value put there to be more verbose.

In fact we do want to keep testing stage 1 here, so I've removed this parameter.

Reply:
One could argue that aarch64 quick is the "similar bot" to aarch64 lld 2 stage, but the goal of this PR was to just port over the missing bits not make further changes.

Might come back to this later.